packages/share/example/ios/RunnerUITests/FLTShareExampleUITests.m

```diff
@@ -0,0 +1,43 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#import <XCTest/XCTest.h>
+
+@interface FLTShareExampleUITests : XCTestCase
+
+@end
+
+@implementation FLTShareExampleUITests
+
+- (void)setUp {
+  self.continueAfterFailure = NO;
+}
+
+- (void)testShareWithEmptyOrigin {
+  XCUIApplication* app = [[XCUIApplication alloc] init];
+  [app launch];
+
+  XCUIElement* shareWithEmptyOriginButton = [app.buttons
+      elementMatchingPredicate:[NSPredicate
+                                   predicateWithFormat:@"label == %@", @"Share With Empty Origin"]];
+  if (![shareWithEmptyOriginButton waitForExistenceWithTimeout:30]) {
+    NSLog(@"%@", app.debugDescription);
```

Comment:
Remove the NSLog

Reply:
This is beneficial so that when a failure happened and we can see what's in the a11y tree and why the particular item was not found in the a11y tree.

Reply:
Can you print it when it fails?  Your test results will get clogged pretty fast with a bunch of debugging lines.

Reply:
Also NSLog is soft deprecated, should be using os_log.

Reply:
The log only happens if `![shareWithEmptyOriginButton waitForExistenceWithTimeout:kSecondsToWaitWhenFindingElements]` and we send a `XCTFail` signal afterwards. So it doesn't print unless the test fails.

Thanks for the info of os_log, updated!

Reply:
Or did you mean to combine the "app.debugDescription" log within XCTFail?